demo-node/package.yaml

```
@@ -63,16 +63,18 @@ examples:
       Use the `where` operator with a more complex expression to identify
       long-running connections in a specific CIDR subnet.
     definition: |
+      // tql2
       export
-      | where #schema == "zeek.conn"
-      | where :ip in 10.10.5.0/25
-      | where orig_bytes > 1 Mi or duration > 30 min
+      where @name == "zeek.conn"
+      legacy "where :ip in 10.10.5.0/25"
+      where orig_bytes > 1 Mi or duration > 30 min
```

```suggestion
      where orig_bytes > 1Mi or duration > 30min
```

This was actually a syntax error.

---

Thanks for the catch! Fixed.